packages/tools/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -1,13 +1,16 @@
 # Changelog
-All notable changes to this widget will be documented in this file.
+All notable changes to this tool will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Increase SVG file size limit in rollup configuration from 16kb to 100kb
```

Comment:
https://github.com/mendix/widgets-resources/blob/01c881855ec8a9186bf67f72a998ead2f3210d62/packages/tools/pluggable-widgets-tools/configs/rollup.config.js#L150

This is also already released

Reply:
Removed